tools/accuracy_checker/accuracy_checker/annotation_converters/fewrel.py

```diff
@@ -0,0 +1,389 @@
+from ._nlp_common import get_tokenizer, CLS_ID, SEP_ID
+
+
+class ERNIEFewrelConverter(BaseFormatConverter):
+    __provider__ = "ernie_fewrel"
+    annotation_types = (TextClassificationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            'testing_file': PathField(description="Path to testing file."),
+            'kgentity_file': PathField(description="Path to knowledge graph entity file."),
+            'vocab_file': PathField(description='Path to vocabulary file.', optional=True),
+            'sentence_piece_model_file': PathField(description='sentence piece model for tokenization', optional=True),
+            'max_seq_length': NumberField(
+                description='The maximum total input sequence length after WordPiece tokenization.',
+                optional=True, default=128, value_type=int
+            ),
+            'max_query_length': NumberField(
+                description='The maximum number of tokens for the question.',
+                optional=True, default=64, value_type=int
+            ),
+            'doc_stride': NumberField(
+                description="When splitting up a long document into chunks, how much stride to take between chunks.",
+                optional=True, default=128, value_type=int
+            ),
+            'lower_case': BoolField(optional=True, default=False, description='Switch tokens to lower case register')
+        })
+
+        return configuration_parameters
+
+    def configure(self):
+        self.testing_file = self.get_value_from_config('testing_file')
+        self.kgentity_file = self.get_value_from_config('kgentity_file')
+        self.max_seq_length = int(self.get_value_from_config('max_seq_length'))
+        self.max_query_length = self.get_value_from_config('max_query_length')
+        self.doc_stride = self.get_value_from_config('doc_stride')
+        self.lower_case = self.get_value_from_config('lower_case')
+        self.tokenizer = get_tokenizer(self.config, self.lower_case)
+        self.support_vocab = 'vocab_file' in self.config
+
+    @staticmethod
+    def _load_examples(file):
+        def _is_whitespace(c):
+            if c == " " or c == "\t" or c == "\r" or c == "\n" or ord(c) == 0x202F:
+                return True
+            return False
+
+        examples = []
+        lines = read_json(file)
+
+        for (i, line) in enumerate(lines):
+            guid = "%s-%s" % ('test', i)
+            for x in line['ents']:
+                if x[1] == 1:
+                    x[1] = 0
+                    #print(line['text'][x[1]:x[2]].encode("utf-8"))
+            text_a = (line['text'], line['ents'])
+            label = line['label']
+            examples.append(
+                {
+                    'id': guid,
+                    'text_a': text_a,
+                    'text_b': None,
```

Comment:
do you really need to put `text_b` if it is always None and never used in this converter?

Reply:
AFAIK there are no text_b in (ERNIE, FewRel) combination